### ISSUE TITLE
game crish while start

### ISSUE THREAD

**Issue by liormalol** [Sunekaer/stoneBlock#424]
## Steps to Reproduce (for bugs)

1.start luncher
2.start game
game crush

## Logs

* Client/Server Log:
* Crash Log: The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
Error: net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Applied Energistics 2 (appliedenergistics2)

## Client Information

* Modpack Version:1.0.23
* Server/LAN/Single Player: single player
* Optifine Installed:
* Shaders Enabled:

**Comment by shortybsd** [comment]
I have the same exact issue with the newest version. Problem is 1 of 2 things. 1. The jar is corrupted for Applied Energistics or 2. There is an issue with a class path in this particular build of Applied Energistics itself.

**Comment by knoxhack** [comment]
Yeah we cant help if you guys dont provite a proper crash-report or log file

**Comment by shortybsd** [comment]
Crash Log: https://pastebin.com/yXr1ivNK

**Comment by Sunekaer** [comment]
could you test with 1.0.24

**Comment by TiffTay** [comment]
Still does not work.
